### ISSUE TITLE
synch Fitbit data when Retrospection is refreshed

### ISSUE THREAD

**Issue by manulita** [sealuzh/PersonalAnalytics#166]

**Comment by s-mueller** [comment]
As @casaout mentioned, this will significantly increase the loading time of the retrospection.

**Comment by casaout** [comment]
> As @casaout mentioned, this will significantly increase the loading time of the retrospection.

Jup, I would strongly suggest against doing so: fetching & processing the fitbit (and in the future possibly other) data would freeze the whole retrospection until the action is completed. This should be done continuously in the background, e.g. once every 20mins as the MsOfficeTracker does it.

@manulita Can we close/delete this issue, or rephrase it to do it continuously?

**Comment by s-mueller** [comment]
@casaout  At the moment we do it every 20 minutes (or whatever time period is used in the Settings.cs file)
